src/images/pillar-blog/2024-09-30-dwmrc-and-ugrc-release-map-to-help-utahns-recycle-better/.placeholder

```diff
@@ -0,0 +1 @@
+delete this file if you are not adding images
```

Comment:
please delete this after adding images or if not using images.

Reply:
Cover image has been uploaded and appears to be working!

Reply:
It's not the correct aspect ratio but we can let that slide.